packages/loader/driver-utils/src/parallelRequests.ts

```diff
@@ -79,7 +79,7 @@ export class ParallelRequests<T> {
                 break;
             }
             this.results.delete(this.nextToDeliver);
-            assert(value.length <= this.payloadSize, "addRequestCore() should break into smaller chunks");
+            assert(value.length <= this.payloadSize, 0x1d9 /* "addRequestCore() should break into smaller chunks" */);
```

Comment:
can you make sure the assert in main gets the same code

Reply:
Pr: https://github.com/microsoft/FluidFramework/pull/5672